contracts/0.4.24/Lido.sol

```diff
@@ -8,16 +8,50 @@ pragma solidity 0.4.24;
 import "@aragon/os/contracts/apps/AragonApp.sol";
 import "@aragon/os/contracts/lib/math/SafeMath.sol";
 
-import "./interfaces/INodeOperatorsRegistry.sol";
-import "./interfaces/ILidoExecutionLayerRewardsVault.sol";
-import "./interfaces/IWithdrawalQueue.sol";
-import "./interfaces/IWithdrawalVault.sol";
-import "./interfaces/IStakingRouter.sol";
-
 import "./lib/StakeLimitUtils.sol";
 
 import "./StETHPermit.sol";
 
+interface ILidoExecutionLayerRewardsVault {
+    function withdrawRewards(uint256 _maxAmount) external returns (uint256 amount);
+}
+
+interface IStakingRouter {
+    function getStakingRewardsDistribution()
+        external
+        view
+        returns (
+            address[] memory recipients,
+            uint256[] memory stakingModuleIds,
+            uint96[] memory stakingModuleFees,
+            uint96 totalFee,
+            uint256 precisionPoints
+        );
+
+    function reportRewardsMinted(uint256[] _stakingModuleIds, uint256[] _totalShares) external;
+
+    function deposit(uint256 maxDepositsCount, uint256 stakingModuleId, bytes depositCalldata) external payable returns (uint256);
+
+    function getWithdrawalCredentials() external view returns (bytes32);
+}
+
+interface IWithdrawalVault {
```

Comment:
Move it closer to  ILidoExecutionLayerRewardsVault?

Reply:
Done